docs/extensions/logging-and-monitoring.md

```diff
@@ -90,31 +97,34 @@ data:
 
 ## Logging
 
-The central fluentd/fluent-bit instances in the `garden` namespace are parsing the logs from all containers in the seed cluster.
-The shoot-specific instances only extract exactly those logs for the pods of the control plane they are responsible for.
-This allows to only fetch the logs for the pods once for the whole cluster, and to distribute them afterwards.
+In Kubernetes clusters, container logs are non-persistent and do not survive stopped and destroyed containers. Gardener addresses this problem for the components that it manages, by introducing its own managed logging solution. It is integrated with the Gardener monitoring stack to have all troubleshooting context in one place.
 
-Extension controllers might deploy components as part of their reconciliation next to the shoot's control plane.
-Examples for this would be a cloud-controller-manager or CSI controller deployments.
-In some cases, the extensions want to submit logging parse configuration for these components such that their logs can be scraped by Gardener's fluentd/fluent-bit deployment(s), and later be visible in the Kibana dashboards.
+!["Cluster Logging Topology"](https://grafana.com/static/assets/img/blog/new-logging-architecture.png?src=tw&mdm=social "Cluster Logging Topology")
```

Comment:
This image is part of #2515, I would prefer to not rely on external resources that we also have in our repo.

Reply:
That was not the case at the time of this writing. As it's been provided now I will link to it.

Reply:
#2515 has been merged.